simulator/cluster_compute_resource_test.go

```diff
@@ -18,13 +18,17 @@ package simulator
 
 import (
 	"context"
+	"path"
 	"testing"
 
+	"github.com/stretchr/testify/require"
 	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
 	"github.com/vmware/govmomi/object"
 	"github.com/vmware/govmomi/simulator/esx"
 	"github.com/vmware/govmomi/simulator/vpx"
 	"github.com/vmware/govmomi/vim25/types"
+	"k8s.io/utils/ptr"
```

Comment:
while I told you to use this on our other project... Because this is a new full import on govmomi just for the pointer check, I would probably not use it and have the old way here (instead of rely on Kubernetes library in this case) as we don't want to add another reference here